### PR TITLE
Fix timezone warning

### LINKED ISSUE #37: TIMEZONE was renamed to TZ in the latest EPICS base
With EPICS 7.0.3.1[1] (I don't know when TZ was introduced.), `TIMEZONE` was renamed to `TZ`. So the iocAdminSoft substitutions [2] file should be updated according to them if one would like to use iocStats vanilla substitutions. The additional changes may be necessary in [3-8].  

[1] https://github.com/epics-base/epics-base/blob/7.0/modules/libcom/src/env/envDefs.h#L66
[2] https://github.com/epics-modules/iocStats/blob/master/iocAdmin/Db/iocAdminSoft.substitutions#L30
[3] https://github.com/epics-modules/iocStats/blob/master/iocAdmin/Db/iocAdminRTEMS.substitutions#L40
[4] https://github.com/epics-modules/iocStats/blob/master/iocAdmin/Db/iocAdminVxWorks.substitutions#L40
[5] https://github.com/epics-modules/iocStats/blob/master/op/ui/autoconvert/ioc_stats_epics_env.ui#L485
[6] op/adl/ioc_stats_epics_env.adl
[7] op/opi/ioc_stats_epics_env.opi
[8] iocAdmin/srcDisplay/ioc_stats_epics_env.edl

### PR DESCRIPTION
**Pull request by mark0n**
This fixes #37.

### CODE REVIEW

**Comment by jeonghanlee** [comment]
Does anyone have a plan to merge this pull request?

**Comment by MarkRivers** [comment]
Why is EPICS_TIMEZONE removed from iocAdminSoft.substitutions and why is EPICS_TZ not added there?

**Comment by jeonghanlee** [comment]
For me, TZ works with typical Soft IOCs with linux-x86_64. However, I saw @anjohnson mentioned that reason.  Please see https://github.com/epics-modules/iocStats/issues/37

**Comment by anjohnson** [comment]
The EPICS_TIMEZONE and EPICS_TZ variables are really only needed on RTEMS or VxWorks IOCs, the workstation OS's use other methods to configure which timezone they should use when displaying a timestamp, so these changes shouldn't really affect anything running on Linux, MacOS or Windows. The iocAdminSoft.substitutions file had a PV showing the EPICS_TIMEZONE variable, but it would never have had any effect on the IOCs loading this file at all so IMHO @mark0n was right to remove it.

I haven't tested these changes myself, has anyone else? As long as someone other than Martin has tried them out I'll be happy to merge this PR. Unfortunately we don't really have anyone who owns this module at the moment so some of the core developers have been working on fixes as we find time, but I think we would prefer that someone else actually takes ownership of it.

**Comment by jeonghanlee** [comment]
@anjohnson  What kind of test do we need to go further? Just running IOC on the Linux is fine? Stress test? Memory usage, and so on? Do we have any standard procedure which each EPICS module can be tested before merging into? 

Let me know, I can help anything which I can do if we go further from this.

**Comment by mark0n** [comment]
@jeonghanlee there are no standard test procedures for EPICS modules and thus their code quality tends to vary. At a minimum someone needs to confirm it compiles without (new) warnings and that my changes actually do what they promise to do. This should already catch most of the stupid mistakes I make everyday and makes it less likely that it only works on my machine. Everything beyond that is up to you - whatever you deem necessary to allow this code to run on your production IOCs. Nobody will blame you if you miss something - it's each lab's responsibility to ensure code they grab from the Internet meets their quality requirements.

File: devIocStats/devIocStatsString.c [hunk 1]
```diff
@@ -274,8 +281,6 @@ static long epics_init_record(stringinRecord* pr)
           ppParam++;
         }
         pr->dpvt = 0;
-        recGblRecordError(S_db_badField,(void*)pr,
-                "devStringinEnvVar (init_record) Illegal INP parm field");
```
Comment (simon-ess) [inline devIocStats/devIocStatsString.c:278]: Why is this being removed? It seems a bit inconsistent now with the other stringin init functions, each of which does print a similar error.

**Review by simon-ess — COMMENTED** [review]
Hi @mark0n - This broadly seems good, one small comment.

**Comment by simon-ess** [comment]
This has been closed in favour of #49, as @mark0n is no longer working with EPICS.

**Pull request closed without merging**